### DISCUSSION TITLE
Strange behaviour on Fedora CoreOS

### DISCUSSION THREAD

**Discussion by RTechSn** [NginxProxyManager/nginx-proxy-manager#2501] [Q&A]
Hi.
I have recently switched to Fedora CoreOS for my projects.
This is my third installation of NPM, and docker-compose file is mostly unchanged, apart from adding :z to mounts, which is required if you have SE Linux enabled.
I'm running into a strange problem that i don't know how to debug.
NPM periodically (at random) stops responding to http(s) requests.
Docker shows that container is running, and container log only shows that everything has started  and ready for connections. 
The moment i do "docker-compose up -d", and it says that it is already up to date, everything works again.
Which way do i look to resolve this?

**Comment by McMac** [comment]
Having the same issue - 4 years later. Running on Ubuntu 24.04.4 LTS. NPM stops responding. Simple container restart helps. Very puzzling…
